fix: preview goes under the menu icon in mobile view

subrataSamartha opened the pull request:
<!-- If your PR fixes an open issue, use `Closes #101` to link your PR with the issue. #101 stands for the issue number you are fixing -->

## 🛠️ Fixes #294 

## 👨‍💻 Changes proposed
Added  `left: 3.2rem;` in box-shadow-preview class of style.css in pages folder.
Removed the previous changes.
Also, changed the title of the pull request.


## ✔️ Check List (Check all the applicable boxes) <!-- Follow the below conventions to check the box -->

- [ x] My code follows the code style of this project.
- [ x] This PR does not contain plagiarized content.
- [ x] The title of my pull request is a short description of the requested changes.


## 📷 Screenshots

<img width="960" alt="image" src="https://user-images.githubusercontent.com/89680196/196954548-a3c7015c-a9dd-432c-8113-ca9c70118222.png">

Dun-sin commented:
Your PR doesn't follow the rules of this project, Follow the rules [👉🏾here👈🏾](https://github.com/Dun-sin/<repo_name>/blob/main/CONTRIBUTING.md)

and make the necessary changes

Dun-sin commented:
Also don't move the box to the center, just move it from under the menu

subrataSamartha commented:
<!-- If your PR fixes an open issue, use `Closes #101` to link your PR with the issue. #101 stands for the issue number you are fixing -->

## 🛠️ Fixes #294 

## 👨‍💻 Changes proposed
Added  `left: 3.2rem;` in box-shadow-preview class of style.css in pages folder.
Removed the previous changes.
Also, changed the title of the pull request.


## ✔️ Check List (Check all the applicable boxes) <!-- Follow the below conventions to check the box -->

- [ x] My code follows the code style of this project.
- [ x] This PR does not contain plagiarized content.
- [ x] The title of my pull request is a short description of the requested changes.


## 📷 Screenshots

<img width="960" alt="image" src="https://user-images.githubusercontent.com/89680196/196954548-a3c7015c-a9dd-432c-8113-ca9c70118222.png">

Dun-sin commented:
> ## 🛠️ Fixes #294
> ## 👨‍💻 Changes proposed
> 
> Added `left: 3.2rem;` in box-shadow-preview class of style.css in pages folder. Removed the previous changes. Also, changed the title of the pull request.
> ## ✔️ Check List (Check all the applicable boxes)
> 
>     * [ x] My code follows the code style of this project.
> 
>     * [ x] This PR does not contain plagiarized content.
> 
>     * [ x] The title of my pull request is a short description of the requested changes.
> 
> 
> ## 📷 Screenshots
> 
> <img alt="image" width="960" src="https://user-images.githubusercontent.com/89680196/196954548-a3c7015c-a9dd-432c-8113-ca9c70118222.png">

you could just edit the old message, doing this was useless

Dun-sin reviewed (approved):
Looks Good 👌🏾 Thank you for contributing your PR is accepted, I hope you continue to contribute to this project


Check out the community discord server [👉🏽here👈🏽](https://discord.com/invite/ufcysW9q23)

Dun-sin commented:
:tada: This PR is included in version 2.21.1 :tada:

The release is available on [GitHub release](https://github.com/Dun-sin/Code-Magic/releases/tag/v2.21.1)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:
